docs/examples/tests/test_mlflow_utilities.py

```diff
@@ -6,48 +6,255 @@
+            """
+            Simulates using the snippets from standalone scripts or the local runtimes.
+            """
+            # Given
+            monkeypatch.setattr(mlflow, "log_metric", Mock())
+            base_uri = "https://foo.example.com"
+            token = "ABC123 mocked token"
+            _mock_config(monkeypatch, base_uri=base_uri, token=token)
 
-        assert Snippets.artifacts_dir_snippet() == tmp_path
+            # When
+            ws_id = Snippets.tracking_uri_token()
 
-    @staticmethod
-    def test_artifact_dir_env_var_override(monkeypatch: MonkeyPatch, tmp_path):
-        monkeypatch.setenv("ORQ_MLFLOW_ARTIFACTS_DIR", str(tmp_path))
-        monkeypatch.setattr(
-            sdk.mlflow._connection_utils, "DEFAULT_TEMP_ARTIFACTS_DIR", "NOT/THIS/PATH"
-        )
+            # Then
+            assert mocked_environ["MLFLOW_TRACKING_URI"] == f"{base_uri}/mlflow/{ws_id}"
+            assert mocked_environ["MLFLOW_TRACKING_TOKEN"] == token
+
+        @staticmethod
+        def test_var_env_override(
+            mocked_environ, monkeypatch, tmp_path, mocked_responses
+        ):
+            """
+            Simulates using the snippets from Studio or CE.
+            """
+            # Given
+            monkeypatch.setattr(mlflow, "log_metric", Mock())
+
+            token = "ABC123 mocked token"
+            passport_path = tmp_path / "passport.txt"
+            passport_path.write_text(token)
+            monkeypatch.setenv("ORQUESTRA_PASSPORT_FILE", str(passport_path))
+
+            cr_name = "mock_cr_name"
+            monkeypatch.setenv("ORQ_MLFLOW_CR_NAME", cr_name)
+
+            mlflow_port = "2037"
+            monkeypatch.setenv("ORQ_MLFLOW_PORT", mlflow_port)
+
+            artifacts_dir = str(tmp_path / "artifacts")
+            monkeypatch.setenv("ORQ_MLFLOW_ARTIFACTS_DIR", artifacts_dir)
+
+            ws_namespace = "mocked-ns"
+            mocked_responses.add(
+                responses.GET,
+                # We're using a regex-based match because the normal URI we send
+                # requests to includes a ZRI and ZRIs are notoriously long.
+                re.compile(
+                    re.escape(
+                        "http://orquestra-resource-catalog.resource-catalog/api/"
+                        "workspaces/"
+                    )
+                ),
+                json={
+                    "namespace": ws_namespace,
+                },
+            )
+
+            # When
+            with pytest.warns(UserWarning):
+                _ = Snippets.tracking_uri_token()
+
+            # Then
+            assert mocked_environ["MLFLOW_TRACKING_TOKEN"] == token
+            assert (
+                mocked_environ["MLFLOW_TRACKING_URI"]
+                == f"http://{cr_name}.{ws_namespace}:{mlflow_port}"
+            )
+
+    class TestTrackingTask:
+        @staticmethod
+        def test_local(mocked_environ, monkeypatch):
```

Comment:
This test fails if config.json doesn't exists. There's a fixture in the main tests' conftest.py that mocks the config file location, we should do something similar here